profiles/student/axk4545.yml

```diff
@@ -0,0 +1,12 @@
+name: Aidan Kahrs
+website: https://axk
+blog: https://axk4545.me
+email: public@axk4545.me
+major: Networking and Systems Administration
+graduation: 2018
```

Comment:
Not required for this PR, but since there are two weeks left in the semester, what do you think about putting this into the `alum/` directory instead?